bundles/binding/org.openhab.binding.panasonictv/src/main/java/org/openhab/binding/panasonictv/PanasonicTVBindingConfig.java

```diff
@@ -0,0 +1,32 @@
+/**
+ * Copyright (c) 2010-2013, openHAB.org and others.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.openhab.binding.panasonictv;
+
+import org.openhab.core.binding.BindingConfig;
+import org.openhab.core.items.Item;
+
```

Comment:
add javadoc briefly explaining the purpose of this class. Please add @author, @since tags